<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:#000;
color:#fff
}
QPushButton{
color:#000;
background-color:rgb(36, 31, 49)
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item alignment="Qt::AlignLeft">
    <widget class="QFrame" name="banner">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="banner_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>1001</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>161</width>
        <height>1006</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="banner_label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>151</width>
         <height>1001</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>0</y>
        <width>901</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>6</y>
         <width>481</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Team Perespective</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>-10</y>
        <width>501</width>
        <height>1031</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color:rgb(97, 53, 131)
}
QPushButton{
background-color:rgb(36, 31, 49);
color:#fff
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>461</width>
         <height>471</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">*{
border-width:25px;
border-color:#fff
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="quit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="icon">
        <iconset theme="application-exit"/>
       </property>
      </widget>
      <widget class="QPushButton" name="select">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Image</string>
       </property>
       <property name="icon">
        <iconset theme="edit-find"/>
       </property>
      </widget>
      <widget class="QPushButton" name="start">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Lets Go</string>
       </property>
       <property name="icon">
        <iconset theme="go-next"/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>720</y>
         <width>311</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Selected Image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>79</y>
        <width>931</width>
        <height>931</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>421</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Junocam Images</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>40</y>
         <width>421</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>480</y>
         <width>421</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>480</y>
         <width>421</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
